### ISSUE TITLE
Are object masks given for the algorithms as inputs?

### ISSUE THREAD

**Issue by daerduoCarey** [xingyuansun/pix3d#6]
Hi, Xingyuan and Jiajun,

Thanks for the great work that provides a great real-world benchmark for 2d-3d reconstruction.

In your paper (https://arxiv.org/pdf/1804.04610.pdf) table 3, you compare the results to PSGN, DRC, MarrNet and AtlasNet, where some use ground-truth object masks as additional inputs and some don't. You said in the paper that "_We use pre-trained models offered by the authors and we crop the input images as required by each algorithm_". I'm wondering if the object masks are used as additional inputs to PSGN and if the comparison in Table 3 fair to all algorithms? Do you use ground-truth object masks to all algorithms?

Thank you!

Bests,
Kaichun

**Comment by xingyuansun** [comment]
Hi Kaichun,

Thank you for asking! For PSGN and AtlasNet, we use ground-truth object masks as additional inputs since they need masks. All other baselines in table 3 do not need masks as input. Thanks!

Best,
Xingyuan
